src/components/Header/Notification.jsx

```diff
@@ -0,0 +1,27 @@
+import { BellIcon } from '@chakra-ui/icons';
+import { IconButton, useColorModeValue } from '@chakra-ui/react';
+import i18next from 'i18next';
+
+export const Notification = () => {
+  const iconsThemeColor = useColorModeValue('teal.900', 'orange.300');
+  const iconsHoverThemeColor = useColorModeValue('teal.900', 'orange.300');
```

Comment:
can we somehow have that on theme level? cause those lines duplicated almost in every component

Reply:
done